tools/build_cros_gcc

```diff
@@ -1,178 +1,210 @@
 #!/bin/bash
 
+ set -e
+
+ # '\033[1;31m' outputs red text. '\033[0m' reverts it to normal.
+ command -v crew &>/dev/null || { echo -e "\033[1;31mcrew command not found.\033[0m"; exit 1; }
+
  GLIBC=glibc-$(crew search glibc -v | tail -1 | cut -d' ' -f2 | cut -d'-' -f1)
- BINUTILS=binutils-2.31.1
+ BINUTILS=binutils-$(crew search binutils -v | tail -1 | cut -d' ' -f2 | cut -d'-' -f1)
  GCC=gcc-8.2.0
```

Comment:
`GCC=$(crew search gcc -v | tail -1 | cut -d' ' -f2 | cut -d'-' -f1)`?  Nothing wrong with gcc 7 and it makes sure when gcc is updated, the script doesn't need modified.  Also, it emulates the current crew build environment more closely which is important for testing purposes.

Reply:
GCC 7 causes errors when cross compiling. Read the GCC 8 ChnageLog.

Reply:
There is no mention of cross-compiling fixes anywhere at https://www.gnu.org/software/gcc/gcc-8/changes.html.  Also, no mentions of cross-compiling bugs at https://gcc.gnu.org/bugzilla/buglist.cgi?bug_status=RESOLVED&resolution=FIXED&target_milestone=7.3.

Reply:
Yeah. Sorry, misleading comment. I encountered issues with the startfiles on GCC 7.